src/components/organisms/HomeContent.tsx

```diff
@@ -51,7 +50,7 @@ interface HomeContentData {
           body: string
           cta: string
           ctaTo: string
-          interactivity: string
+          interactivity: 'image'
```

Comment:
suggestion(blocking): It seems like we now only support `'image'` "interactivity". I believe this justifies a refactoring of the component(s) here. We can basically utilize the same image loading via graphql as in `siteMetadata` (meaning relative path inside the content `.json`)
We could then get rid of the `interactiveComponents` object and handle rendering again in the `paragraphs.map()` completely.